changelog/7667.feature.rst

```diff
@@ -0,0 +1 @@
+New `--durations-min` command-line flag controls the minimal duration for inclusion in the slowest list of tests shown by `--durations`. Previously this was hard-coded to `0.005s`.
```

Comment:
It's RST, double backticks should be used

Reply:
Done.

Reply:
> It's RST, double backticks should be used

You are correct, but we changed our sphinx configuration to consider single ticks as monospaced font, so either way is OK for the docs/changelog entries.